src/cli.ts

```diff
@@ -274,4 +275,20 @@ program
     }
   });
 
+// TOTP command
+program
+  .command('totp <secret>')
+  .description('Generate a Time-based One-Time token using the provided secret.')
+  .option('--issuer <issuer>', 'Provider or Service the secret is associated with.')
+  .option('--label <label>', 'Account Identifier (default: SyntheticsTOTP)')
+  .action((secret, cmdOpts: TOTPCmdOptions) => {
+    try {
+      const token = totp(secret, cmdOpts)
+      write(bold(`OTP Token: ${token}`));
+    } catch (e) {
+      e && error(e);
```

Comment:
What is the benefit of filtering out the falsy values in this case? It might be confusing if the program exits on error but no error is reported

Reply:
I'd rather we print the error, whatever that might end up being
```suggestion
      error(e);
```